### PR TITLE
Exercises for weeks 1 and 2

### PR DESCRIPTION
**Pull request by linajandren**
Sorry for being late with a pull request. First two weeks of exercises here, working on third week.

### CODE REVIEW

File: andren-lina/interpreter/interpreter.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    assert False, f"Unknown variable {name}"
+
+def env_set(env, name, value):
+    assert isinstance(name, str)
+    env[name] = value # lägg till i den sista i chainmap-en
+    
+
+# dictionary of all operations defined
+
+#Change this to adding to a chain map? and change environment functions to work for chain map. finally, main must send empty chainmap. is that it? No, also need to update do_call
```
Comment (gvwilson) [inline andren-lina/interpreter/interpreter.py:122]: ;+1:

File: andren-lina/interpreter/test-program.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[
+    "seq",
+    ["set", "alpha", 1],
+    ["set", "beta", 2],
+    ["add", ["get", "alpha"], ["get", "beta"]]
```
Comment (gvwilson) [inline andren-lina/interpreter/test-program.json:5]: :+1:

File: andren-lina/tester/counting-results-1.py [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                func()
+                results[name] = 'pass'
+        except AssertionError as e:
+            if kind == "fail":
+                results[name] = 'pass'
+            else: 
+                results[name] = 'fail'
+        except Exception as e:
+            results[name] = 'error - ' + str(e)
+    print('fail: '+ str(len([x for x in results.values() if x=='fail']))+'\n'
```
Comment (gvwilson) [inline andren-lina/tester/counting-results-1.py:52]: :+1: I would probably write a helper function that takes `results` and a key like `'fail'` in order to shorten these lines for readability

**Review by gvwilson — APPROVED** [review]

**Comment by linajandren** [comment]
I've added persistence exercises with strings and with globals(). Haven't finished the aliasing exercise so it isn't here yet.

**Pull request merged**